axelor-human-resource/src/main/java/com/axelor/apps/hr/web/expense/ExpenseLineController.java

```diff
@@ -82,4 +83,14 @@ public void computeProjectTaskDomain(ActionRequest request, ActionResponse respo
     String domain = Beans.get(ExpenseLineService.class).computeProjectTaskDomain(expenseLine);
     response.setAttr("projectTask", "domain", domain);
   }
+
+  public void setInvitedCollaboratorSetDomain(ActionRequest request, ActionResponse response) {
+    ExpenseLine expenseLine = request.getContext().asType(ExpenseLine.class);
+    String domain =
+        "self.id IN ("
+            + StringHelper.getIdListString(
+                Beans.get(ExpenseLineService.class).getEmployeeDomain(expenseLine.getExpenseDate()))
+            + ")";
```

Comment:
This code should be placed in a Service like ExpenseLineDomainService with a method name related to invited collaborators.